# Roomba State configuration
# Everything in mks units, all angles in radians, unless otherwise stated

# Rate to check for new messages
roomba_state_estimator_update_rate: 20

# how long (in seconds) 
# to wait on a roomba message
roomba_state_estimator_start_up_timeout: 20

# speeds below this indicate
# the rooomba is turning
turning_detected_speed: .10

# speeds above this indicate
# the roomba is moving forward
moving_detected_speed: .25

# expected velocity for a roomba
roomba_expected_vel: .30
